rubricscore.cpp

```diff
@@ -299,6 +307,13 @@ void print_score(const rubric_score& the_score) {
             << std::endl;
 
   std::cout << line << std::endl << std::endl;
+
+  // Add final results and maximum possible points to JSON file
+  resultJSON.addFinalResult(total_earned_points, total_possible_points);
+
+  // Generate JSON file to local directory
+  std::string filename = "resultOutput.json";
```

Comment:
Parameters like this filename should be declared prominently as constants. Please create a constant at the top right after the #includes, like

const std::string RESULT_JSON_FILENAME("resultOutput.json");